config/webpack.config.base.js

```diff
@@ -48,6 +48,17 @@ module.exports = {
     new HtmlWebpackPlugin({
       template: path.resolve(__dirname, '../src/index.ejs'),
       title: pkg.name,
+      excludeChunks: ['services'],
+      inject: 'head',
+      minify: {
+        collapseWhitespace: true
+      }
+    }),
+    new HtmlWebpackPlugin({
+      template: path.resolve(__dirname, '../src/services.ejs'),
+      title: pkg.name,
```

Comment:
I think we should just remove the title for the moment, as having it means that we need to care about i18n. Since the iframe is embedded in another page, I do not think this title will be used.